django_app/redbox_app/redbox_core/models.py

```diff
@@ -295,6 +300,14 @@ def delete_from_s3(self):
         """Manually deletes the file from S3 storage."""
         self.original_file.delete(save=False)
 
+    def delete_from_elastic(self):
+        index = f"{env.elastic_root_index}-chunk"
+        if es_client.indices.exists(index=index):
+            es_client.delete_by_query(
+                index=index,
+                body={"query": {"term": {"metadata.file_name.keyword": self.unique_name}}},
```

Comment:
Will this delete files with this name for all users? If two users upload the same file and one deletes it, they'll both go?